torchvision/transforms/functional_tensor.py

```diff
@@ -885,7 +885,10 @@ def autocontrast(img: Tensor) -> Tensor:
 
 
 def _scale_channel(img_chan):
-    hist = torch.histc(img_chan.to(torch.float32), bins=256, min=0, max=255)
+    if img_chan.is_cuda:
+        hist = torch.histc(img_chan.to(torch.float32), bins=256, min=0, max=255)
+    else:
+        hist = torch.bincount(img_chan.view(-1), minlength=256)
```

Comment:
Do you need to clip values between 0 and 255 to keep things consistent between cuda/cpu?

Reply:
Passing `minlength=256` ensures that `bincount` will count the number of values for all values in `[0, 255]`, so `histc` and `bincount` would be equivalent.

In the `histc` call we have `bins=256, min=0, max=255`, which IIUC assumes that the image is within `[0, 255]` already, is this not the case?